### PR TITLE
[Test] Add a dataset shuffle test

### PR DESCRIPTION
**Pull request by rkooo567**
## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: release/nightly_tests/dataset/shuffle_ingest.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    ray.get(tasks)
+
+
+if __name__ == "__main__":
+    parser = create_parser()
+    args = parser.parse_args()
+    print("Connecting to Ray cluster...")
+    ray.init(address=args.address)
+
+    run_consume(args, data_dir=None)
```
Comment (scv119) [inline release/nightly_tests/dataset/shuffle_ingest.py:78]: should we write the test result into the env TEST_OUTPUT file?

Comment (rkooo567) [reply release/nightly_tests/dataset/shuffle_ingest.py:78]: oh good catch. I will fix that.

File: release/nightly_tests/dataset/shuffle_ingest.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    "embeddings_name16": 73699,
+    "one_hot0": 1,
+    "one_hot1": 30,
+    "labels": 0.3801173847541949,
+    "__index_level_0__": 0
+}
+
+
+def create_parser():
+    parser = argparse.ArgumentParser(description="Eric Example")
```
Comment (richardliaw) [inline release/nightly_tests/dataset/shuffle_ingest.py:32]: rename this description to maybe "Test script to evaluate distributed shuffling with consumers".

File: release/nightly_tests/dataset/shuffle_ingest.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    parser.add_argument("--address")
+    parser.add_argument("--num-rows", type=int, default=8 * (10**8))
+    parser.add_argument("--num-files", type=int, default=50)
+    parser.add_argument("--read-cache", action="store_true", default=False)
```
Comment (richardliaw) [inline release/nightly_tests/dataset/shuffle_ingest.py:36]: remove this?

File: release/nightly_tests/dataset/shuffle_ingest.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    parser.add_argument(
+        "--data-dir",
+        type=str,
+        default="s3://shuffling-data-loader-benchmarks/data/")
```
Comment (richardliaw) [inline release/nightly_tests/dataset/shuffle_ingest.py:40]: we can remove this too

**Comment by rkooo567** [comment]
Any of test failure should be related. It only changes the test file.

File: release/nightly_tests/dataset/shuffle_ingest.py (hunk 1 above)
Comment (scv119) [reply release/nightly_tests/dataset/shuffle_ingest.py:78]: image

**Review by scv119 — APPROVED** [review]
i'll trust you won't forget to add TEST_OUTPUT

**Comment by rkooo567** [comment]
Oh I haven’t pushed changes lol.

**Comment by rkooo567** [comment]
Preferred over @scv119 's change which includes more logic

**Pull request closed without merging**